packages/scaffold-config/src/frameworks.ts

```diff
@@ -61,6 +61,22 @@ function getBundlerDependency (bundler: WizardBundler['type'], projectPath: stri
   }
 }
 
+export const WIZARD_MOUNT_MODULES = ['cypress/react', 'cypress/react18', 'cypress/vue', 'cypress/vue2', 'cypress/angular'] as const
+
+export type WizardMountModule = typeof WIZARD_MOUNT_MODULES[number]
+
+const mountModule = (mountModule: WizardMountModule) => async (projectRoot: string) => Promise.resolve(mountModule)
+
+const reactMountModule = async (projectPath: string) => {
```

Comment:
/nitpick: Could we standardize these function signatures to both use `projectRoot` or `projectPath`?

Reply:
Went with `projectPath` to make it consistent: [143a9d1](https://github.com/cypress-io/cypress/pull/23251/commits/143a9d171d0389a749eac7907842e2d59390a6cf)